web/src/lib/components/shared-components/search-bar/search-camera-section.svelte

```diff
@@ -0,0 +1,63 @@
+<script lang="ts" context="module">
+  export interface SearchCameraFilter {
+    make?: string;
+    model?: string;
+  }
+</script>
+
+<script lang="ts">
+  import { SearchSuggestionType, getSearchSuggestions } from '@immich/sdk';
+  import Combobox, { toComboBoxOptions } from '../combobox.svelte';
+
+  export let filters: SearchCameraFilter;
+
+  let makes: string[] = [];
+  let models: string[] = [];
+
+  $: makeFilter = filters.make;
+  $: modelFilter = filters.model;
+  $: updateMakes(modelFilter);
+  $: updateModels(makeFilter);
```

Comment:
This will have to be updated when https://github.com/immich-app/immich/pull/7382 is merged since it's calling an `async` function without error handling. I see the same in other files below